corehq/apps/app_manager/templates/app_manager/form_view.html

```diff
@@ -56,7 +56,7 @@
   </style>
 
   {% compress js %}
-    <script src="{% static 'cloudcare/js/formplayer/utils/util.js' %}"></script>
```

Comment:
>utils/util

Finally a compromise position between the "util" and "utils" camps!

Reply:
😭